dts/bindings/xen,domu.yaml

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2024 EPAM Systems
+# SPDX-License-Identifier: Apache-2.0
+
+description: Xen DomU configuration
+compatible: "xen,domu"
+
+include: base.yaml
+
+properties:
+  reg:
+    required: true
+    description: Domain ID
+
+  machine_dt_compat:
+    type: string
+    required: true
+
+  mem_size:
+    type: int
```

Comment:
Just want to clarify: does this type allows values longer than 32 bits?

Reply:
I'm not sure, but I suppose this type only supports 32-bit values. Thank you for pointing this out.